src/window/mod.rs

```diff
@@ -258,7 +258,15 @@ impl<F: Frame + 'static> Window<F> {
                         // Check whether we should save old size for later restoration.
                         let should_stash_size = states
                             .iter()
-                            .find(|s| *s == &State::Maximized || *s == &State::Fullscreen)
+                            .find(|s| match *s {
+                                State::Maximized
+                                | State::Fullscreen
```

Comment:
It's done that way instead of inversion of states, since the `match` is non exhaustive.